ignite/metrics/rouge.py

```diff
@@ -59,15 +58,15 @@ def lcs(seq_a: Sequence[Any], seq_b: Sequence[Any]) -> int:
     return dp[m][n]
 
 
-class Score(namedtuple("Score", ["match", "candidate", "reference"])):
+class _Score(namedtuple("_Score", ["match", "candidate", "reference"])):
```

Comment:
I don't understand why you pass this argument in private. It is used as output of public functions.

Reply:
Either we make it private or provide documentation for it. If you want I can write docs for it.